config/routes.rb

```diff
@@ -4,5 +4,7 @@
   ActiveAdmin.routes(self)
   root to: 'movies#index'
   resources :aws_texts
-  resources :questions
+  resources :questions do
+    resource :solutions
+  end
```

Comment:
必要なアクションに限定するのが理想です。他も設定されていないので，よければ修正しておいて下さい。

```suggestion
  resources :questions, only: [:index, :create, :show] do
    resource :solutions, only: :create
  end
```